### PR TITLE
EN-12608: add read_from_nbe=true and version=2.1 to query

### PR DESCRIPTION
**Pull request by NathalieKaligirwa**
- add `read_from_nbe` and `version` params on core queries
- update tests

### CODE REVIEW

File: src/test/scala/com.socrata.tileserver/util/GeoProviderTest.scala [hunk 1]
```diff
@@ -40,12 +39,12 @@ class GeoProviderTest extends TestBase with UnusedSugar with MockitoSugar {
...
         addHeader("X-Socrata-Host" -> "geo.provider.test").
         addHeader(knownHeader).
-        addParameters(augmentedWithQueryTimeout).
+        addParameters(augmented).
         get.builder
 
       val client = testcommon.mocks.StaticCuratedClient { request =>
         val actual = request(base).builder
-
+
```
Comment (khjrtbrg) [inline src/test/scala/com.socrata.tileserver/util/GeoProviderTest.scala]: Nit: remove the extra whitespace here?

File: src/main/scala/com.socrata.tileserver/util/GeoProvider.scala [hunk 2]
```diff
@@ -25,8 +25,6 @@ case class GeoProvider(client: CuratedServiceClient) {
   def doQuery(info: RequestInfo): GeoResponse = {
     val intersects = filter(info.tile, info.geoColumn, info.overscan.getOrElse(0))
     val params = augmentParams(info, intersects)
-    val paramsWithTimeout = addQueryTimeout(params, config.TileServerConfig.queryTimeout)
```
Comment (khjrtbrg) [inline src/main/scala/com.socrata.tileserver/util/GeoProvider.scala]: We don't need `config.TileServerConfig.queryTimeout` anymore in our request? Or is it getting in there some other way?

Comment (khjrtbrg) [inline src/main/scala/com.socrata.tileserver/util/GeoProvider.scala]: 🔍  aha! Thanks! ✨

**Comment by chitang** [comment]
I wonder if it makes sense to put $$version and $$read_from_nbe into config file.  I mean under they key like additional_paramaters.  But it is fine if you don't think that is a good idea.

**Comment by NathalieKaligirwa** [comment]
@chitang I think adding two extra parameters in the code is fine for now. A better refactor, would be to have a config file for the `GeoProvider` that contains all the keys.

**Comment by chitang** [comment]
ok

**Comment by khjrtbrg** [comment]
Demoed it works locally and that it doesn't appear to change functionality for non-derived view feature maps ✨

**Comment by chitang** [comment]
Actually, I think we should try not to expose $$read_from_nbe (and to a lesser degree $$version=2.1) to users.

**Comment by chitang** [comment]
:shipit:

**Pull request merged**